### ISSUE TITLE
custom vscode snippets not loading

### ISSUE THREAD

**Issue by boydkelly** [L3MON4D3/LuaSnip#707]
I seem to have the identical issue as reported by several users here:
https://www.reddit.com/r/neovim/comments/w28ktb/custom_snippets_not_working_luasnip/

Since several are reporting this issue is it a bug?   Or is there a configuration problem?

I have converted utisnips to vscode snippets and they are located in ~/.config/nvim/snippets.   
I can use friendly-snippets just fine but not my custom converted snippets.
I have set the path for custom snippets to ./snippets as well and also only used the first require with no path as according the the docs here it seems that should be sufficient.  I tried both lazy_load and load.

As I test I *moved* the html.json snippet file from the installed cache and put it in my custom ./snippets directory and the snippets are not available.

````
 {    
  | "L3MON4D3/LuaSnip",    
  | dependencies = {    
  | ¦ "rafamadriz/friendly-snippets",    
  | ¦ config = function()    
  | ¦ ┆ require("luasnip.loaders.from_vscode").lazy_load()    
  | ¦ ┆ require("luasnip.loaders.from_vscode").load({ paths = "~/.config/nvim/snippets" })                                     
  | ¦ end,    
  | },    
  }    
````

**Comment by L3MON4D3** [comment]
I think that's a layer-8-error, has been almost every time so far :P
Everything you wrote sounds correct though, turn on logging (eg. turn it to `"info"`) (only on master, in case you have some version pinned), that should tell you what is going on/wrong.
See [doc](https://github.com/L3MON4D3/LuaSnip/blob/master/DOC.md#logging) for some details on handling the log.

**Comment by boydkelly** [comment]
Thanks for pointing me to the logs!   I've got i working.
I have a yaml.json and asciidoc.json in my snippets directory.  If I open a yaml file I get the message:

````
 ERROR | vscode-loader: Tried reading package /var/home/bkelly/.config/nvim/snippets/package.json, but it does not exist
````
I concocted my own package.json file based on what I found in the friendly-snippets package with just yaml and asciidoc.   And this solves my issue.   Maybe the snippet-converter should have either created that or warned that it is a requirement.   Anyway I got it going.
